src/main/java/dev/pgm/events/format/score/Scores.java

```diff
@@ -44,16 +44,8 @@ public FormattedScore formattedScore(TournamentTeamManager teamManager) {
       // size == 2, therefore display two way score
       Pair<Score, Score> topTwo = topTwoOrdered();
       return new TwoTeamFormattedScore(teamManager, topTwo, updated);
-      // return teamManager.teamColour(topTwo.first.team()) + topTwo.first.team().teamName() + " " +
-      // topTwo.first.score() + ChatColor.GRAY + " - " +
-      // teamManager.teamColour(topTwo.second.team()) + topTwo.second.score() + " " +
-      // topTwo.second.team().teamName();
     }
 
-    // size != 2, therefore return just the top score
```

Comment:
Probably good to leave this line here. This bit is the only thing that really has to be implement (from what I remember) to support more than two teams.